.github/workflows/publish.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish @wasm-miniscript
+on:
+  workflow_dispatch:
+   push:
+     branches:
+       - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  publish:
+    name: Publish Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+          override: true
+
+      - name: Install wasm-pack
+        run: |
+          cargo install wasm-pack
```

Comment:
@rushilbg Ideally this dependency would be in a `Cargo.toml` file in order to lock the version and the associated checksum.